core/controllers/skill_editor.py

```diff
@@ -141,6 +141,47 @@ def get(self, skill_id):
 class EditableSkillDataHandler(base.BaseHandler):
     """A data handler for skills which supports writing."""
 
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'version': {
+                'schema': {
+                    'type': 'int'
+                }
+            },
+            'commit_message': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': constants.MAX_COMMIT_MESSAGE_LENGTH
+                    }]
+                }
+            },
+            'change_dicts': {
+                'schema': {
+                    'type': 'list',
+                    'items': {
+                        'type': 'object_dict',
+                        'object_class': skill_domain.SkillChange
+                    }
+                }
+            }
+        },
+        'DELETE': {}
+    }
+
+    URL_PATH_ARGS_SCHEMAS = {
+        'skill_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_valid_skill_id'
+                }]
```

Comment:
Can you `ENTITY_ID_REGEX` use regex here?

Reply:
Yes I will add it